index.html

```diff
@@ -12,67 +12,206 @@
     font-size: 12px;
     color: #111111;
 }
-#solIcon {
-    right: 10px;
-    top: 10px;
+#editor {
     position: absolute;
-    height: 100px;
-    width: 100px;
-    overflow: hidden;
-}
-#solIcon img {
-    top: -20px;
-    right: -10px;
-    position: absolute;
-    height: 100px;
-}
-#optimizeBox {
-    position: absolute;
-    top: 30px;
-    left: 720px;
-    font-size: 14px;
-}
-#input {
-    position: absolute;
-    top: 120px;
+    top: 0;
     left: 0px;
-    width: 700px;
+    width: auto;
     bottom: 0px;
+    right: 37em;
+    
+}
+#input {
     font-size: 15px;
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    min-width: 20vw;
 }
-#output {
+
+#righthand-panel {
     position: absolute;
-    top: 120px;
-    left: 720px;
-    right: 10px;
+    top: 0;
+    width: 37em;
```

Comment:
Also worth noting is that the 37em applies to the output panel on the right, and the editor would (in the default case take up the rest of the space)